### PR TITLE
UMCS-319 - Fehler in Java SDK beim Parsen des Datums

### PR DESCRIPTION
**Pull request by DerAusHeidelberg**
- Changed Date-Formatting to be parsed and formatted with UTC as Base and the correct time shift

### CODE REVIEW

File: src/test/java/com/unzer/payment/business/AbstractPaymentTest.java [hunk 1]
```diff
@@ -449,7 +452,10 @@ protected Metadata getTestMetadata(boolean sorted) {
     }
 
     protected Date getDate(String date) throws ParseException {
-        return new SimpleDateFormat("dd.MM.yy").parse(date);
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate parsedLocalDate = LocalDate.parse(date, formatter);
+        return Date.from(parsedLocalDate.atStartOfDay().atZone(ZoneId.of("UTC")).toInstant());
+        //return new SimpleDateFormat("dd.MM.yy").parse(date);
```
Comment (Ryouzanpaku) [inline src/test/java/com/unzer/payment/business/AbstractPaymentTest.java:458]: please remove comment.

**Review by Ryouzanpaku — COMMENTED** [review]
Looking good so far, but I am missing dedicated test for this change.

File: src/test/java/com/unzer/payment/business/AbstractPaymentTest.java (hunk 1 above)
Comment (DerAusHeidelberg) [reply src/test/java/com/unzer/payment/business/AbstractPaymentTest.java]: removed.

**Comment by DerAusHeidelberg** [comment]
Tests were added.

**Review by Ryouzanpaku — APPROVED** [review]
LGTM

**Pull request merged**